README.md

```diff
@@ -1,8 +1,64 @@
-# Jarvis Data Engineering Training
-1. [Linux cluster monitoring agent (Linux and SQL)](./linux_sql) In-progress
-2. [Core Java Apps](./core_java) In-progress
-3. [Springboot Trading REST API](./springboot) In-progress
-4. [Hadoop/Hive](./hadoop) In-progress
-5. [Spark/Scala](./spark) In-progress
-6. [Cloud/DevOps](./cloud_devops) In-progress
+# David Gordon . Jarvis Consulting
 
+Hi, my name is David Gordon. I am a professional Software Engineer based in Toronto, working on various projects at Jarvis. I'm experienced in various technologies, languages and frameworks, creating efficient applications and elegant interfaces.
```

Comment:
This is very short and sweet, but there can be more oomph to it. Feel free to give a couple more lines and list out some of the technologies or experience you have to support your claims.